### PR TITLE
Change timeout configurations to accept decimals

### PR DESCRIPTION
**Pull request by Bhashinee**
## Purpose
Fixes the: https://github.com/ballerina-platform/ballerina-standard-library/issues/1024

### CODE REVIEW

**Comment by Bhashinee** [comment]
Passing a decimal value to a remote function gives the error
`error: jVM generation is not supported for type decimal`. Hence did not change `int` in close function to `decimal`

Raised issue: https://github.com/ballerina-platform/ballerina-lang/issues/28959

File: websocket-ballerina/websocket_async_client.bal [hunk 1]
```diff
@@ -216,27 +216,27 @@ public type ClientConfiguration record {|
 #
 # + subProtocols - Negotiable sub protocols of the client
 # + customHeaders - Custom headers, which should be sent to the server
-# + idleTimeoutInSeconds - Idle timeout of the client. Upon timeout, the `onIdleTimeout` resource (if defined)
+# + idleTimeout - Idle timeout of the client. Upon timeout, the `onIdleTimeout` resource (if defined)
```
Comment (ldclakmal) [inline websocket-ballerina/websocket_async_client.bal:219]:
```suggestion
# + idleTimeout - Idle timeout (in seconds) of the client. Upon timeout, the `onIdleTimeout` resource (if defined)
```

File: websocket-ballerina/websocket_async_client.bal [hunk 2]
```diff
@@ -216,27 +216,27 @@ public type ClientConfiguration record {|
...
 #                          of the client service will be triggered
-# + readTimeoutInSeconds - Read timeout of the client. This is applicable only for the Sync client
+# + readTimeout - Read timeout of the client. This is applicable only for the Sync client
```
Comment (ldclakmal) [inline websocket-ballerina/websocket_async_client.bal:221]:
```suggestion
# + readTimeout - Read timeout (in seconds) of the client. This is applicable only for the Sync client
```

File: websocket-ballerina/websocket_caller.bal [hunk 3]
```diff
@@ -69,15 +69,15 @@ public client class Caller {
     #
     # + statusCode - Status code for closing the connection
     # + reason - Reason for closing the connection
-    # + timeoutInSeconds - Time to wait for the close frame to be received from the remote endpoint before closing the
+    # + timeout - Time to wait for the close frame to be received from the remote endpoint before closing the
```
Comment (ldclakmal) [inline websocket-ballerina/websocket_caller.bal:72]:
```suggestion
    # + timeout - Time to wait (in seconds) for the close frame to be received from the remote endpoint before closing the
```

File: websocket-ballerina/websocket_failover_client.bal [hunk 4]
```diff
@@ -80,15 +80,15 @@ public client class WebSocketFailoverClient {
     #
     # + statusCode - Status code for closing the connection
     # + reason - Reason for closing the connection
-    # + timeoutInSeconds - Time to wait for the close frame to be received from the remote endpoint before closing the
+    # + timeout - Time to wait for the close frame to be received from the remote endpoint before closing the
```
Comment (ldclakmal) [inline websocket-ballerina/websocket_failover_client.bal:83]:
```suggestion
    # + timeout - Time to wait (in seconds) for the close frame to be received from the remote endpoint before closing the
```

File: websocket-ballerina/websocket_sync_client.bal [hunk 5]
```diff
@@ -101,15 +99,15 @@ public client class Client {
     #
     # + statusCode - Status code for closing the connection
     # + reason - Reason for closing the connection
-    # + timeoutInSeconds - Time to wait for the close frame to be received from the remote endpoint before closing the
+    # + timeout - Time to wait for the close frame to be received from the remote endpoint before closing the
```
Comment (ldclakmal) [inline websocket-ballerina/websocket_sync_client.bal:102]:
```suggestion
    # + timeout - Time to wait (in seconds) for the close frame to be received from the remote endpoint before closing the
```

**Review by ldclakmal — APPROVED** [review]

**Pull request merged**